### PR TITLE
Added field collections to Thng

### PR DESCRIPTION
**Pull request by sollecitom**

### CODE REVIEW

File: thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java [hunk 1]
```diff
@@ -110,4 +117,14 @@ public String firstIdentifier() {
 
 		return identifiers.values().iterator().next();
 	}
+
+	public Set<String> getCollections() {
+
+		return collections != null ? new TreeSet<>(collections) : null;
```
Comment (dorogush) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java]: Why not Collections.unmodifiableSet?

Comment (sollecitom) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java]: For I need ordering.

michele sollecito ////
back-end developer
m +44 7463 268226 /// w evrythng.com

Comment (dorogush) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java]: Collections.unmodifiableSortedSet ?

Comment (sollecitom) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java]: With regards to unmodifiableSortedSet:

1) It returns a SortedSet which, apparently, is not assignable to a Set;
2) I don't want to forbid people to add stuff there (e.g., tests or subsequent calls), I just want to preserve encapsulation.

**Pull request merged**